### ISSUE TITLE
Character processing can fail on custom sorting lists

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#222]
The loop over the Char sorting list is encountering undefined keys.

```
mule.js:1270 Uncaught TypeError: Cannot read property 'Equipment' of undefined
    at Mule.parse (mule.js:1270)
    at Mule.query (mule.js:496)
    at Object.callback (muledump.js:1565)
    at HTMLDivElement.<anonymous> (lightbox.js:476)
    at HTMLDivElement.dispatch (jquery.min.js:2)
    at HTMLDivElement.y.handle (jquery.min.js:2)

**Comment by jakcodex** [comment]
Dead characters are not being accounted for properly it seems.

**Comment by jakcodex** [comment]
Released in 9.4.
